charts/galoy/values.yaml

```diff
@@ -628,8 +631,76 @@ oathkeeper:
+    mutatorIdTokenJWKs: "http://galoy-oathkeeper-api:4456/.well-known/jwks.json"
+    accessRules: |
+      [
+        {
+          "id": "decline-direct-access-validatetoken",
+          "upstream": { "url": "http://api:4002" },
+          "match": {
+            "url": "<(http|https)>://<[a-zA-Z0-9-.:]+>/auth/validatetoken",
+            "methods": ["GET", "POST"]
+          },
+          "authenticators": [ { "handler": "unauthorized" } ],
+          "authorizer": { "handler": "allow" },
+          "mutators": [ { "handler": "noop" } ]
+        },
+        {
+          "id": "allow-clear-cookies-endpoint",
+          "upstream": { "url": "http://api:4002" },
+          "match": {
+            "url": "<(http|https)>://<[a-zA-Z0-9-.:]+>/auth/clearCookies",
+            "methods": ["GET", "POST", "OPTIONS"]
+          },
+          "authenticators": [ { "handler": "anonymous" } ],
+          "authorizer": { "handler": "allow" },
+          "mutators": [ { "handler": "noop" } ]
+        },
+        {
+          "id": "galoy-backend",
+          "upstream": {
+            "url": "http://api:4002"
+          },
+          "match": {
+            "url": "<(http|https)>://<[a-zA-Z0-9-.:]+>/<(?!auth/clearCookies)(.*)>",
```

Comment:
adding the matcher to exclude the `auth/clearCookies` endpoint. if this is not set then it has two matches and won't resolve. this endpoint needs to be able to pass expired or revoked cookies so the server can clear them for the browser and avoid login issues. this still allows all wildcard endpoints excluding `auth/clearCookies`